addons/l10n_ro_edi/models/account_move.py

```diff
@@ -54,59 +58,75 @@ def _l10n_ro_edi_create_attachment_values(self, raw, res_model=None, res_id=None
         res_model = res_model or self._name
         res_id = res_id or self.id
         return {
-            'name': f"ciusro_{self.name.replace('/', '_')}.xml",
+            'name': f"ciusro_signature_{self.name.replace('/', '_')}.xml",
             'res_model': res_model,
             'res_id': res_id,
             'raw': raw,
             'type': 'binary',
             'mimetype': 'application/xml',
         }
 
-    def _l10n_ro_edi_create_document_invoice_sending(self, key_loading, attachment_raw):
-        """ Shorthand for creating a `l10n_ro_edi.document` of state `invoice_sending` """
+    def _l10n_ro_edi_create_document_invoice_sending(self, values: dict):
```

Comment:
Is there a reason to this change? 
Couldn't we instead of changing the parameter for a dict, keep the function definition as it and unpack the dict?

Reply:
It doesn't really have to change, but I did it for 2 reasons:
- to make it the same for all `_l10n_ro_edi_create_document_*` helper functions
- so that it is easier to fix something in stable (if for any reason we want to pass more data, we can just include it through the dictionary instead of changing the function signature)

Reply:
Since when are e against dynamic typing?

Reply:
Function methods should be clear still. The attachment_raw and key_values have clearly different purposes, so they are logical in the params.